wasm/src/libc++/minimal.cc

```diff
@@ -9,18 +9,10 @@ void* operator new(size_t size) {
     return opa_malloc(size);
 }
 
-void* operator new(unsigned long size, std::align_val_t) {
```

Comment:
Some context in the commit message on why these can be deleted now would be nice to have.

Reply:
✔️ Good point